modules/apps/jaggery/mdm/src/mdm/units/user-update/public/js/bottomJs.js

```diff
@@ -24,6 +24,10 @@ $(document).ready(function () {
     $("select.select2[multiple=multiple]").select2({
         tags : true
     });
+    var roleList = $("#roles").attr("selectedVals").trim().replace(/ /g,"");
+    roleList = roleList.replace(/(\r\n|\n|\r)/gm,"");
```

Comment:
Recommended fixes  can be found in PR[1]

[1]. https://github.com/wso2/product-mdm/pull/158